app/client/mindjogg/components/StdButton/StdButton.test.js

```diff
@@ -0,0 +1,37 @@
+import StdButton from "./StdButton";
+import React from "react";
+import renderer from "react-test-renderer";
+//import { fireEvent, render } from "@testing-library/react-native";
+
+describe("Standard Text Button", () => {
+  it("Should Render", () => {
+    const onClickCallback = jest.fn();
+
+    const tree = renderer
+      .create(
+        <StdButton
+          text={"I am a Test"}
+          buttonColour={"#9B7FBA"}
+          buttonWidth={80}
+          buttonHeight={80}
+          buttonPress={onClickCallback}
+        />
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+});
+
+// describe("Standard Text Button", () => {
```

Comment:
I think you should copy/paste this test for longPress as well

Reply:
Ah I can do that in a later PR 👍, will need to make an issue for it

Reply:
wicked sounds good